server/app/views/applicant/ProgramIndexView.java

```diff
@@ -286,41 +286,37 @@ private LiTag programCard(
               preferredLocale, cardData.latestSubmittedApplicationStatus().get()));
     }
     programData.with(title, description);
-
-    // Add info link.
-    String infoUrl =
-        controllers.applicant.routes.ApplicantProgramsController.view(applicantId, program.id())
-            .url();
-    ATag infoLink =
-        new LinkElement()
-            .setId(baseId + "-info-link")
-            .setStyles("mb-2", "text-sm", "underline")
-            .setText(messages.at(MessageKey.LINK_PROGRAM_DETAILS.getKeyName()))
-            .setHref(infoUrl)
-            .asAnchorText()
-            .attr(
-                "aria-label",
-                messages.at(
-                    MessageKey.LINK_PROGRAM_DETAILS_SR.getKeyName(),
-                    program.localizedName().getOrDefault(preferredLocale)));
-    programData.with(div(infoLink));
-
-    // Add external link if it is set.
+    // Add external link if it is set else info link.
     if (!program.externalLink().isEmpty()) {
       ATag externalLink =
           new LinkElement()
               .setId(baseId + "-external-link")
               .setStyles("mb-2", "text-sm", "underline")
-              .setText(messages.at(MessageKey.EXTERNAL_LINK.getKeyName()))
+              .setText(messages.at(MessageKey.LINK_PROGRAM_DETAILS.getKeyName()))
               .setHref(program.externalLink())
               .opensInNewTab()
               .setIcon(Icons.OPEN_IN_NEW, LinkElement.IconPosition.END)
               .asAnchorText()
-              .attr(
-                  "aria-label",
-                  messages.at(MessageKey.EXTERNAL_LINK_OPENS_IN_NEW_TAB.getKeyName()));
+              .attr("aria-label", messages.at(MessageKey.LINK_PROGRAM_DETAILS_SR.getKeyName()));
```

Comment:
**suggestion (blocking):** This message string includes a [message](https://sourcegraph.com/github.com/civiform/civiform@5de389d/-/blob/server/conf/messages?L152) about what program it's for, so change to `messages.at(MessageKey.LINK_PROGRAM_DETAILS_SR.getKeyName(), program.localizedName().getOrDefault(preferredLocale))` or change to LINK_PROGRAM_DETAILS.

Reply:
Got rid of this block. We don't need external link anymore. Confuses the residents.